docs/design/scheduler_object_states.md

```diff
@@ -52,8 +52,10 @@ Containers/pods can be added to and removed from the application.
 This state is part of the normal scheduling cycle.
 * Waiting: An application that has no pending requests or running containers/pod will be waiting.
 This state shows that the application has not been marked completed yet but currently is not actively being scheduled.
-* Completed: The resource manager has signalled that the application is done. 
-This is a final state. The application cannot change state after entering.
+* Completed: An application is considered completed when it has been in the waiting state for a defined time period.
```

Comment:
We should add that a completed application can only move to expired and cannot move back into a scheduling state like running or waiting.